uiplib/gui/mainGui.py

```diff
@@ -90,7 +84,6 @@ def set_wallpaper(self):
 
     def download(self):
         """Method to start download."""
-        pass
```

Comment:
I made the change because PyUnusedCodeBear asked me to and it made sense to me, even if the function is a stub, we dont need a `pass` statement.
So should I remove the button and the `stub` or leave it without a `pass`